src/components/Viewport.ts

```diff
@@ -384,11 +384,26 @@ export default class Viewport {
   }
 
   public enable(): void {
-    this.panInput.enable();
+    if (!this.panInput) {
+      const options = this.options;
+      this.panInput = new PanInput(this.viewportElement, {
+        inputType: options.inputType,
+        thresholdAngle: options.thresholdAngle,
+        iOSEdgeSwipeThreshold: options.iOSEdgeSwipeThreshold,
+        scale: options.horizontal ? [-1, 0] : [0, -1],
+      });
+
+      this.axes.connect(options.horizontal ? ["flick", ""] : ["", "flick"], this.panInput);
```

Comment:
@WoodNeck

It probably overlaps with the init code.

Reply:
Thanks for checking, I've updated my code